### DISCUSSION TITLE
Integrating react-hook-form with ui5-webcomponents-react/FilterBar

### DISCUSSION THREAD

**Discussion by apoorvsemwal** [react-hook-form/react-hook-form#6160] [Q&A]
Hello, contributors at react-hook-form,

We have been trying to use RHF with [ui5-webcomponents-react](https://github.com/SAP/ui5-webcomponents-react)

There is a particular scenario where we are not able to achieve the expected behavior when using 
ui5-webcomponents-react [FilterBar](https://github.com/SAP/ui5-webcomponents-react/blob/8607c0319c82ca25db6ac8d560d8db32c85c8e7a/packages/main/src/components/FilterBar/index.tsx#L196) and [Input](https://github.com/SAP/ui5-webcomponents-react/blob/8607c0319c82ca25db6ac8d560d8db32c85c8e7a/packages/main/src/webComponents/Input/index.tsx#L157) with react-hook-form.

I have shared the exact details of the issue with a codesandbox setup at:
[Integrating react-hook-form with webcomponents-react/FilterBar](https://github.com/SAP/ui5-webcomponents-react/issues/1890)

I would appreciate any guidance on making ui5-webcomponents-react FilterBar work with react-hook-form.

Thanks

**Comment by majidzadeh** [comment]
Thank you for posting this.

This is impacting our code and delivery as well. When can we expect to get a response on this?

Regards,
Khalid
